src/apps/rDSN.monitor/app_package/MonitorApp.py

```diff
@@ -449,11 +449,6 @@ def post(self):
             savedFile.write(raw_file)
             savedFile.close()
             
-            #TODO: shall we need some mechanism to detect if 7z are truely installed?
```

Comment:
this is unfortunately necessary on our app store so that we can easily navigate the internal content of the package from the web site.